### PR TITLE
database ddl

### PR DESCRIPTION
**Pull request by Ansper11**
Hi this is our database ddl and test cases please check

### CODE REVIEW

File: Database/MC_EDUDatabase.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+LessonID int NOT NULL,
+MobName varchar(30),
+NumOfMob int,
+
+primary key (Uname,MobID,LessonID),
+FOREIGN key(Uname) REFERENCES User (Name)
+);
+
+
+INSERT INTO user VALUES("player1","password1","aaa@ubc.ca");
```
Comment (Nick-McGee) [inline Database/MC_EDUDatabase.sql:51]: May have to rework this, as I'm not sure if it's a secure way of storing user info.

**Comment by eloiseespel** [comment]
So happy to see that slowly coming to life!

**Comment by eloiseespel** [comment]
I think it might be good to have some visualization data of that just for ourselves

**Comment by yyht6300a** [comment]
It is a good start.

**Pull request merged**